src/main/java/com/es/phoneshop/model/entity/cart/CartService.java

```diff
@@ -0,0 +1,10 @@
+package com.es.phoneshop.model.entity.cart;
```

Comment:
Move all service related classes in package called **service**. Put this package on the same level as **dao** and **entity**. In service package create sub-package called **impl** there will be all you service implementation classes